src/community/security/oauth2-openid-connect/oauth2-openid-connect-core/src/main/java/org/geoserver/security/oauth2/OpenIdConnectFilterConfig.java

```diff
@@ -100,4 +100,12 @@ protected StringBuilder buildAuthorizationUrl() {
             sb.append("&response_mode=").append(responseMode);
         return sb;
     }
+
+    protected StringBuilder buildEndSessionUrl(final String idToken) {
```

Comment:
Found documentation about it here: 
https://openid.net/specs/openid-connect-rpinitiated-1_0.html#RPLogout

It says recommended, however in other places (e.g., [Okta](https://developer.okta.com/docs/reference/api/oidc/#logout)) it was documented as mandatory.

So I guess it's a good thing to have :-D

Reply:
Yep, currently I would say this is **strongly** recommended, in most of the cases **mandatory**.

In any case having this won't break the protocol.